server/configuration.go

```diff
@@ -70,6 +75,21 @@ func (p *Plugin) setConfiguration(configuration *configuration) {
 
 // OnConfigurationChange is invoked when configuration changes may have been made.
 func (p *Plugin) OnConfigurationChange() error {
+	// Set defaults
+	dirty := false
+	mapCfg := p.API.GetPluginConfig()
+
+	if v, ok := mapCfg["appid"]; !ok || v == "" {
```

Comment:
What's this for?

Reply:
Just setting a default value for app id on first run (or if it is empty).